data/transport missions.txt

```diff
@@ -3469,3 +3469,105 @@ mission "Inheritance Redirection"
+			`	"Oxygenpockets were as much a feature on my home-planet as mountains and woodlands are on more habitable planets."`
+			label "help request"
+			`	Vanessa pauses for a moment and then continues, "We're testing the VI's cargo refrigeration system using the Testament, a modified Hauler III. This system is one of our biggest achievements. To show how well it works, we plan to throw a formal dinner on <destination> with fresh sushi transported from here. I'm sure everyone at the dinner will be amazed by how well the food has been preserved despite the travel time. I'd like you to escort the ship and its cargo to <destination>."`
+			choice
+				`	"Isn't it a bit risky to transport food using a refrigeration system that hasn't been tested yet?"`
+				`	"Am I also invited for the dinner?"`
+					goto invited
+				`	"Why do you need me to escort the ship?"`
+					goto internal
+				`	"Sure, I'll help you."`
+					accept
+			`	"Risky, why? I don't see what can go wrong. The system is quite well designed. I'm so proud of my team."`
+			choice
+				`	"Am I also invited for the dinner?"`
+				`	"Why do you need me to escort the ship?`
+					goto internal
+			label "invited"
+			`	"I'm sorry, but you are not invited. This dinner is only for Southbound Shipyards top-level management and representatives of the <destination> government."`
+			label "internal"
+			`	"Normally we'd look internally to satisfy any escort needs, that's true. But we've discovered a mole in our organization and there's very strong competition from Syndicated Shipyards. Going externally is safer in this case, and you came recommended when we searched through the list of captains available on short notice. You will be paid well for your time. Will you help us?"`
+			choice
+				`	"Sure, I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+	on visit
+		dialog `You land on <destination>, but the Testament hasn't caught up with you yet. Better take off and wait for it to appear in this system.`
+	on complete
+		dialog
+			`You wait for the Testament to land, but it remains in orbit.`
+			`There is no response to your hails. You need to take off and investigate.`
+
+mission "Hauler VI cargo prototype 2"
+	name "Escort Hauler prototype to a streetfood festival"
+	description "Escort the Testament to <destination> to deliver cooked fish dishes to a streetfood festival."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Hauler VI cargo prototype 1: done"	
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on enter
+		dialog
+			`As soon as you take off, you receive an angry voice message from Vanessa over a short-range inter-ship communications channel: "Why were you trying to contact me? Does everyone need to know that we're here?"`
```

Comment:
So this line makes sound like the player didn't even contact Vanessa, have only just launched, so perhaps say something like "You take off and try to contact Vanessa, but receive..."

The description "a short-range inter-ship communications channel" isn't used anywhere else, so perhaps just drop this part entirely (receive an angry voice message from Vanessa: "Why...").
The game also doesn't tend to distinguish between voice and text or video messages, so perhaps just say "an angry message."